spec/migrations/20180823111741_move_location_led_state_to_asset_details_table_spec.rb

```diff
@@ -0,0 +1,51 @@
+require_migration
+
+describe MoveLocationLedStateToAssetDetailsTable do
+  let(:physical_server_stub)  { migration_stub(:PhysicalServer) }
+  let(:physical_chassis_stub) { migration_stub(:PhysicalChassis) }
+  let(:asset_detail_stub)     { migration_stub(:AssetDetail) }
+
+  migration_context :up do
+    it "should migrate the data from physical_chassis and physical_servers to asset_details table" do
+      asset_detail_server = asset_detail_stub.create!
+      physical_server_stub.create!(
+        :asset_detail       => asset_detail_server,
+        :location_led_state => 'Off'
+      )
+
+      asset_detail_chassis = asset_detail_stub.create!
+      physical_chassis_stub.create!(
+        :asset_detail       => asset_detail_chassis,
+        :location_led_state => 'Blinking'
+      )
+
+      migrate
+
+      expect(asset_detail_server.reload.location_led_state).to  eq('Off')
+      expect(asset_detail_chassis.reload.location_led_state).to eq('Blinking')
+    end
+  end
```

Comment:
I think there should also be a test where there is a `PhysicalServer` and `PhysicalChassis` and no `AssetDetails`.

Reply:
(and same for down)